oak_runtime/src/node/invocation.rs

```diff
@@ -0,0 +1,142 @@
+//
```

Comment:
Almost entirely copied from `oak_runtime/src/node/grpc/invocation.rs`. It is used both in `HttpClientNode` and `GrpcClientNode`.